src/xb-main.c

```diff
@@ -185,6 +186,42 @@ server_get_fix_callback (GHashTable *params,
     }
 }
 
+/* GET /test - test for existence of a particular feature
+ * Returns:
+ *     200 - Feature supported by this instance of xapian-bridge
+ *     400 - The required "feature" parameter wasn't specified
+ *     404 - Feature not supported by this instance of xapian-bridge
+ *
+ * Note: xapian-bridge versions predating the addition of /test return 500
+ * (default response to an unknown URL) so you typically want to handle 500
+ * in the same way as 404.
+ */
+static void
+server_get_test_callback (GHashTable *params,
+                          GHashTable *query,
+                          SoupMessage *message,
+                          gpointer user_data)
+{
+  XapianBridge *xb = user_data;
+  const char *feature;
+
+  feature = g_hash_table_lookup (query, "feature");
+  if (feature == NULL)
+    {
+      server_send_response (message, SOUP_STATUS_BAD_REQUEST, NULL, NULL);
+      return;
+    }
+
+  if (strcmp (feature, "query-param-defaultOp") == 0)
+    {
```

Comment:
Thanks, though that really shouldn't be a private URL.  It also doesn't say to avoid `//` comments!